.qube.yml

```
@@ -0,0 +1,24 @@
+qube_version: '2.6.1 # <<QUBE_NO_BUMP>>'
+full_name: Sven Fillinger
+email: sven.fillinger@qbic.uni-tuebingen.de
+project_name: ukt-diagnostics
+project_short_description: ukt-diagnostics. A qube based .
+version: 1.3.0-SNAPSHOT
```

```suggestion
version: 1.3.0
```
Since this is intended for a release i'd remove the SNAPSHOT from the version 
I'd recommend the Qube bump-version functionality for this purpose.